cpp/doxygen/Doxyfile

```diff
@@ -1162,7 +1162,7 @@ HTML_STYLESHEET        =
 # list). For an example see the documentation.
 # This tag requires that the tag GENERATE_HTML is set to YES.
 
-HTML_EXTRA_STYLESHEET  = rapids.css
+HTML_EXTRA_STYLESHEET  =
```

Comment:
You will also need to set footer:
```
HTML_FOOTER = footer.html
```

Reply:
In cuspatial this property doesn't seem was set

Reply:
I see, we probably will have to add that entry to let doxygen know to add the file to footer of all pages.

Reply:
I was looking somewhere else. Fixed.